services/datamanager/builtin/sync_test.go

```diff
@@ -332,6 +332,7 @@ func TestArbitraryFileUpload(t *testing.T) {
 	datasync.RetryExponentialFactor.Store(int32(1))
 	fileName := "some_file_name.txt"
 	fileExt := ".txt"
+	emptyFileTestName := "error due to empty file, local files should not be deleted"
```

Comment:
why not define this in line like all of the other test names?

Reply:
main reason was just because when I do the special casing in the actual test logic, i didn't want the check to be hardcoded (`if tc.name == "error due to empty file, local files should not be deleted"`) in case the test name needed to be changed for any reason. 

if you think it's better to hard code it though, i'm definitely open to changing it!

Reply:
i think thats a good reason!